pyiron_atomistics/vasp/base.py

```diff
@@ -395,6 +396,22 @@ def collect_output(self):
         except (IOError, ValueError, FileNotFoundError):
             pass
 
+        # Bader analysis
+        if os.path.isfile(self.working_directory + "/AECCAR0") and os.path.isfile(self.working_directory + "/AECCAR2"):
+            bader = Bader(self)
+            try:
+                charges_orig, volumes_orig = bader.call_bader_from_job()
+                charges, volumes = charges_orig.copy(), volumes_orig.copy()
+                charges[self.sorted_indices] = charges_orig
+                volumes[self.sorted_indices] = volumes_orig
+                if "valence_charges" in self._output_parser.generic_output.dft_log_dict.keys():
+                    valence_charges = self._output_parser.generic_output.dft_log_dict["valence_charges"]
+                    # Positive values indicate electron depletion
+                    self._output_parser.generic_output.dft_log_dict["bader_charges"] = valence_charges - charges
+                self._output_parser.generic_output.dft_log_dict["bader_volumes"] = volumes
+            except ValueError:
```

Comment:
I have the feeling this `try` and `except` statement is a bit lengthy. Possible errors could be changes in the Bader code which lead to different output files or simply the Bader executable not being available on a given system, but both would result in the same error message making it hard to debug.

Reply:
Thanks @niklassiemer, I've fixed this!